src/test/java/uk/gov/hmcts/ccd/endpoint/std/CallbackTest.java

```diff
@@ -1405,7 +1403,6 @@ public void shouldReturn404WhenPostCreateEventWithInvalidEventTokenForCitizen()
 
     @Test
     @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = {"classpath:sql/insert_callback_cases.sql"})
-    @Transactional
```

Comment:
are we sure @Transactional is not required here?

Reply:
Not completely sure, but with it left in the test consistently fails with the following postgres error

`ERROR: canceling statement due to user request`

I have tried playing with timeouts, but without much luck

Reply:
Ok, if the test passes repeatedly without that annotation then we're good.